pipelines/short-read-alignment-vector/farm5/input_files/AN0131-C.json

```diff
@@ -1,21 +1,7 @@
 {
   "ShortReadAlignment.sample_id": "AN0131-C",
   "ShortReadAlignment.output_basename": "AN0131-C",
-  "ShortReadAlignment.read_group_ids": [
-    "HS22_09812:4",
-    "HS16_10209:3",
-    "HS16_10209:4"
-  ],
-  "ShortReadAlignment.input_fastq1s": [
-    "/lustre/scratch118/malaria/team112/personal/gg18/ag1000g-phase1-minimal/ERR317337_1.fastq.gz",
-    "/lustre/scratch118/malaria/team112/personal/gg18/ag1000g-phase1-minimal/ERR340933_1.fastq.gz",
-    "/lustre/scratch118/malaria/team112/personal/gg18/ag1000g-phase1-minimal/ERR340945_1.fastq.gz"
-  ],
-  "ShortReadAlignment.input_fastq2s": [
-    "/lustre/scratch118/malaria/team112/personal/gg18/ag1000g-phase1-minimal/ERR317337_2.fastq.gz",
-    "/lustre/scratch118/malaria/team112/personal/gg18/ag1000g-phase1-minimal/ERR340933_2.fastq.gz",
-    "/lustre/scratch118/malaria/team112/personal/gg18/ag1000g-phase1-minimal/ERR340945_2.fastq.gz"
-  ],
+  "ShortReadAlignment.input_file": "/lustre/scratch118/malaria/team112/personal/gg18/ag1000g-phase1-minimal/lanelets.tsv",
```

Comment:
Can we include an example of the input file?

Reply:
I'll provide an example in the readme's